SafeguardSessions/TROUBLESHOOTING.md

```diff
@@ -17,9 +18,18 @@ To see if there were any error during the data import, you can check the **Info*
 
 If the error is handled by the Power BI Connector, the pop-up will contain the error code and error message what you can find in [Errors raised by the Power BI Connector] section to learn more about the error.

```

Comment:
What do we mean here by "handled by"? 
a, The error originates from the Power BI Connector's side
b, The connector will be able to handle the error
c, The connector automatically handles the issue, and this is just an FYI message for the user? 
d, Other  :-)

Reply:
I'll look for an updated version of this in the next PR. :-)